schema/partner_show.js

```diff
@@ -98,6 +98,26 @@ const PartnerShowType = new GraphQLObjectType({
         gravity(`partner/${partner.id}/show/${id}`)
           .then(({ events }) => events),
     },
+    counts: {
+      type: new GraphQLObjectType({
+        name: 'PartnerShowCounts',
+        fields: {
+          artworks: {
+            type: GraphQLInt,
+            args: {
+              artist_id: {
+                type: GraphQLString,
+                description: 'The slug or ID of an artist in the show.',
+              },
```

Comment:
Yeah it probably needs to be edited to accept that, but it's fine—you'll just have to process it to add commas or whatever else you wanna do on your client in the meantime.